sdk/scheduler/src/main/java/com/mesosphere/sdk/offer/evaluate/PortEvaluationStage.java

```diff
@@ -115,16 +114,6 @@ protected void setProtos(PodInfoBuilder podInfoBuilder, Protos.Resource resource
         }
     }
 
-    protected Protos.Resource getFulfilledResource(Protos.Resource resource) {
```

Comment:
Yes.  In fact it's for the `setResourceId` callback that it mostly needs to still extend.  That and `super.setProtos`.  I could make the `setResourceId` method part of an interface that they both implement instead, and then factor out `setProtos`.